.travis.yml

```diff
@@ -3,21 +3,20 @@ language: python
 cache: pip
 
 python:
-  - "2.7"
   - "3.6"
   - "3.7"
   - "3.8"
   - "3.9"
   - "3.10"
   - "3.11"
-  - "pypy"
+  - "3.12"
   - "pypy3"
 
 install:
   - python -m pip install coverage .[dev]
 
 script:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2* || $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then coverage run --source=python2 -m unittest discover python2/tests; fi
+  # - if [[ $TRAVIS_PYTHON_VERSION == 2* || $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then coverage run --source=python2 -m unittest discover python2/tests; fi
```

Comment:
Let's just totally clean up this Python 2 stuff. It will still be there in version 4.4!